packages/client/hmi-client/src/services/provenance.ts

```diff
@@ -28,16 +29,19 @@ const MAX_RELATED_ARTIFACT_COUNT = 5;
 /**
  For a document, find similar content
 	Find related documents (xDD)
-*/
+ */
 
 // API helper function for fetching provenance data
 async function getConnectedNodes(
-	id: string | number,
+	id: string,
 	rootType: ProvenanceType
 ): Promise<ProvenanceResult | null> {
+	const publication: DocumentAsset | null = await getDocument(id);
+	if (!publication) return null;
+
 	// FIXME: all underscore naming should be fixed
```

Comment:
Does this FIXME still necessary, has it been fixed now?

Reply:
This hasn't been fixed yet:
```
const body: ProvenanceQueryParam = {
		root_id: Number(id),
		root_id: publication.id,
		root_type: rootType
		root_type: rootType
	};
	};
```
The corresponding java object is also the same.
I've opened a new issue to clean up the underscores and move this to the TS generator: https://github.com/DARPA-ASKEM/Terarium/issues/849